video/out/mac/common.swift

```diff
@@ -104,6 +104,12 @@ class Common: NSObject {
         window.setMinimized(minimized)
         window.makeMain()
         window.makeKey()
+        window.isMovable = !forcePosition
```

Comment:
are you misunderstanding the option here (https://github.com/mpv-player/mpv/blob/master/DOCS/man/options.rst?plain=1#L3569-L3572)?

> Forcefully move mpv's video output window to default location whenever
    there is a change in video parameters, video stream or file. This used to
    be the default behavior. Currently only affects X11 and SDL VOs.

this just prevents window dragging on title bar and background.

i believe the only part missing here is the the position on init. though i have to check what exactly didn't work properly.